contracts/modules/cover/CoverMigrator.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: GPL-3.0-only
 
-pragma solidity ^0.8.9;
+pragma solidity ^0.8.16;
```

Comment:
I would like a comment on this file that reminds us this contract will replace 

https://github.com/NexusMutual/smart-contracts/blob/master/contracts/modules/claims/Claims.sol#L82


And the signature of submitClaims needs to match exactly for compatibility.